static/app/views/replays/detail/console/messageFormatter.spec.tsx

```diff
@@ -117,7 +159,7 @@ describe('MessageFormatter', () => {
   it('Should print console message correctly when it is an Error object', () => {
     render(<MessageFormatter breadcrumb={breadcrumbs[2]} />);
 
-    expect(screen.getByText('Error: this is my error message')).toBeInTheDocument();
+    expect(screen.getByText('{}')).toBeInTheDocument();
```

Comment:
does it still make sense to have this test? weird that the UI will render `{}` - its not very informative

Reply:
Good call, I re-added the special case for this